ENIGMAsystem/SHELL/Widget_Systems/KDialog/dialogs.cpp

```diff
@@ -196,6 +192,20 @@ void show_debug_message(string errortext, MESSAGE_TYPE type) {
   if (strtod(str_result.c_str(), NULL) == 1) exit(0);
 }
 
+namespace enigma_user {
+
+void show_debug_message(string errortext, MESSAGE_TYPE type) {
+  if (type != M_INFO && type != M_WARNING) {
+    #ifndef DEBUG_MODE
```

Comment:
Let the caller decide if they want special handling for debug mode, Sam. People will want to know that the graphics system failed to initialize, even outside of debug mode.